src/olympia/stats/views.py

```diff
@@ -355,12 +356,15 @@ def stats_report(request, addon, report, beta=False):
     stats_base_url = reverse('stats.overview.beta' if beta else
                              'stats.overview', args=[addon.slug])
     view = get_report_view(request)
+    hide_beta = addon.type not in [ADDON_EXTENSION, ADDON_STATICTHEME]
+
     return render(
         request,
         'stats/reports/%s.html' % report,
         {
             'addon': addon,
             'beta': beta,
+            'hide_beta': hide_beta,
```

Comment:
`hide_beta` and `link_to_old_stats` are the same, right?  Does a common variable name make sense?

Reply:
yeah, they are but they convey different intents so that's why I used two different names :/